client/src/featureform/resources.py

```diff
@@ -578,20 +578,24 @@ def _create(self, stub) -> None:
         stub.CreateTrainingSetVariant(serialized)
 
     def _create_local(self, db) -> None:
+        self._insert_training_set_features(db)
```

Comment:
Whats this for?

Reply:
This stores the feature name and the training associated with it in a separate table so that it is easy to access the corresponding features